front/app/containers/Admin/pagesAndMenu/containers/CustomPages/New/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CustomPageSettingsForm from '../CustomPageSettingsForm';
+import { Box } from '@citizenlab/cl2-component-library';
+import Breadcrumbs from 'components/UI/Breadcrumbs';
+import { pagesAndMenuBreadcrumb } from 'containers/Admin/pagesAndMenu/breadcrumbs';
+import HelmetIntl from 'components/HelmetIntl';
+import messages from '../messages';
+import { InjectedIntlProps } from 'react-intl';
+import { injectIntl } from 'utils/cl-intl';
+
+const CustomPagesNewSettings = ({
+  intl: { formatMessage },
+}: InjectedIntlProps) => {
+  return (
+    <>
+      <HelmetIntl
+        title={messages.newCustomPageMetaTitle}
+        description={messages.newCustomPageMetaDescription}
```

Comment:
Why do you need the description on an Admin page? Isn't it used only for SEO? 
Also, I haven't found any `Helmet` occurrences in `front/app/containers/Admin/*.ts*`

Reply:
Agreed, we don't need the description here. We don't have it in admin for now, but we should/will standardise the title for all admin pages. So it should have a title (as it's visible).

Also discussed with Ben in #dev-flexible-pages.